add markup example to tfoot

scottaohara opened the pull request:
Fixes #1356 

Adds a `table` with `tfoot` as an example for its section.

scottaohara committed:
add example to tfoot

add a table with `tfoot` as an example for the section.

chaals commented on sections/semantics-tabular-data.include:
```diff
@@ -1075,6 +1075,52 @@
+            <td>+$50</td>
+          </tr>
+        </tbody>
+        <tfoot>
```
Do you want to `<mark>` the tfoot here? It's a big example...

(that means switching to use pre... so you may not.

scottaohara replied:
mmm...it is big... maybe i cut out some of the `tr`s instead...and people will just have to assume my math is correct?

otherwise, yeh.  Probably makes sense to use `mark` and `pre`.  If you don't like the above mentioned idea, @chaals, i'll move to `pre`.

chaals replied:
Either way. I have a minor preference for `mark`ing things in examples. Pik one, do it, and feel free to merge when it builds clean.

scottaohara committed:
mark tfoot

convert xmp to pre for example. escape < and >
delete a row of content and adjust math in tfoot
wrap tfoot with mark element
